Overview.bs

```diff
@@ -1688,11 +1688,55 @@ guidance that encoder implementations may want to consider.
 
 <b>Utilize an existing font subsetter implementation</b>
 
-<!-- TODO: A high quality encoder should preserve functionality as discussed above. Existing font subsetters can be used to assist
-     constructing encodings that meet this requirement. They provide all of the logic needed to preserve functionality. High quality
-     open source subsetters exist -->
+As discussed in [[#encoder]] a high quality encoder should preserve the functionality of the original font. One way to
+achieve this is to leverage an existing font subsetter implementation to produce font subsets that retain the functionality
+of the original font. The IFT patches can then be derived from these subsets.
+
+A font subsetter produces a [=font subset=] from an input font based on a desired [=font subset definition=]. The practice of reliably
+subsetting a font is well understood and has multiple open-source implementations (a full formal description is
+beyond the scope of this document). It typically involves a reachability analysis, where the data in tables is examined
+relative to the font subset definition to see which portions can be reached by any possible content covered by the subset definition.
+Any reachable data is retained in the generated font subset, while any unreachable data may be removed.
+
+Example pseudo code for a very basic encoder implementation using a font subsetter:
+
+```
+# Encodes a font (full_font) into an incremental font that starts at base_subset_def
+# and can incrementally add any of subset_definitions. Returns the IFT encoded font
+# and set of associated patches.
+encode_as_ift(full_font, base_subset_def, subset_definitions):
+  base_font = subset(full_font, base_subset_def)
+  patches = encode_node(full_font, base_font, base_subset_def, subset_definitions)
```

Comment:
You could make this

```
patches, base_font = encode_node(full_font, base_font, base_subset_def, subset_definitions)
```

to make it clearer that the base_font is modified by encode_node

Reply:
Done.